### ISSUE TITLE
508 contrast to small on secondary nav links

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#363]
Both the current and red non-current tabs have insufficient contrast.

>
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.94:1. Recommendation: change text colour to #767676.

![screen shot 2016-06-16 at 11 28 21 am](https://cloud.githubusercontent.com/assets/1701077/16128364/4f1439b0-33b5-11e6-8867-07f6e0782552.png)

![screen shot 2016-06-16 at 2 07 33 pm](https://cloud.githubusercontent.com/assets/1701077/16133233/9118d24c-33cb-11e6-9e7c-7f47bac3f373.png)

**Comment by msecret** [comment]
@thisisdano the light gray for links on the subnav is too light compared to the white background. Can you suggest a darker color? You can check http://webaim.org/resources/contrastchecker/ to see what would be compliant

**Comment by msecret** [comment]
Merged into #374
